fluss-common/src/test/java/org/apache/fluss/row/columnar/ColumnarMapTest.java

```diff
@@ -0,0 +1,338 @@
+                .hasMessageContaining("Key array size and value array size must be equal");
+    }
+
+    @Test
+    public void testSize() {
+        Object[] keys = {
+            BinaryString.fromString("a"),
+            BinaryString.fromString("b"),
+            BinaryString.fromString("c"),
+            BinaryString.fromString("d")
+        };
+        Object[] values = {1, 2, 3, 4};
+
+        InternalArray keyArray = new GenericArray(keys);
+        InternalArray valueArray = new GenericArray(values);
+
+        ColumnarMap map = new ColumnarMap(keyArray, valueArray);
+
+        assertThat(map.size()).isEqualTo(4);
+    }
+
+    @Test
+    public void testEmptyMap() {
+        Object[] keys = {};
+        Object[] values = {};
+
+        InternalArray keyArray = new GenericArray(keys);
+        InternalArray valueArray = new GenericArray(values);
+
+        ColumnarMap map = new ColumnarMap(keyArray, valueArray);
+
+        assertThat(map.size()).isEqualTo(0);
+        assertThat(map.keyArray().size()).isEqualTo(0);
+        assertThat(map.valueArray().size()).isEqualTo(0);
+    }
+
+    @Test
+    public void testKeyArrayAndValueArray() {
+        Object[] keys = {
+            BinaryString.fromString("name"),
+            BinaryString.fromString("age"),
+            BinaryString.fromString("city")
+        };
+        Object[] values = {BinaryString.fromString("Alice"), 30, BinaryString.fromString("NYC")};
+
+        InternalArray keyArray = new GenericArray(keys);
+        InternalArray valueArray = new GenericArray(values);
+
+        ColumnarMap map = new ColumnarMap(keyArray, valueArray);
+
+        assertThat(map.keyArray()).isSameAs(keyArray);
+        assertThat(map.valueArray()).isSameAs(valueArray);
+
+        assertThat(map.keyArray().getString(0)).isEqualTo(BinaryString.fromString("name"));
+        assertThat(map.valueArray().getString(0)).isEqualTo(BinaryString.fromString("Alice"));
+        assertThat(map.keyArray().getString(1)).isEqualTo(BinaryString.fromString("age"));
+        assertThat(map.valueArray().getInt(1)).isEqualTo(30);
+    }
+
+    @Test
+    public void testEquals() {
+        Object[] keys1 = {BinaryString.fromString("k1"), BinaryString.fromString("k2")};
+        Object[] values1 = {1, 2};
+        InternalArray keyArray1 = new GenericArray(keys1);
+        InternalArray valueArray1 = new GenericArray(values1);
+        ColumnarMap map1 = new ColumnarMap(keyArray1, valueArray1);
+
+        Object[] keys2 = {BinaryString.fromString("k1"), BinaryString.fromString("k2")};
+        Object[] values2 = {1, 2};
+        InternalArray keyArray2 = new GenericArray(keys2);
+        InternalArray valueArray2 = new GenericArray(values2);
+        ColumnarMap map2 = new ColumnarMap(keyArray2, valueArray2);
+
+        assertThat(map1.equals(map1)).isTrue();
+        assertThat(map1.equals(map2)).isTrue();
+        assertThat(map2.equals(map1)).isTrue();
+    }
+
+    @Test
+    public void testNotEquals() {
```

Comment:
remove all the tests about `equals`, `toString` and `hashcode`. Tests don't need to cover these logics. If we want to improve coverage, they should has already been invoked in other tests and thus has test coverage.